CourseGrab/Home/HomeTableViewCell.swift

```diff
@@ -8,7 +8,152 @@
+        container.clipsToBounds = false
+        container.layer.cornerRadius = 5
+        container.layer.shadowColor = UIColor.black.cgColor
+        container.layer.shadowOpacity = 0.3
+        container.layer.shadowRadius = 2
+        container.layer.shadowOffset = .zero
+        container.backgroundColor = .white
+        contentView.addSubview(container)
+        
+        courseLabel.translatesAutoresizingMaskIntoConstraints = false
+        courseLabel.font = .boldSystemFont(ofSize: 14)
+        courseLabel.textColor = .courseGrabGray
+        container.addSubview(courseLabel)
+        
+        enrollButton.translatesAutoresizingMaskIntoConstraints = false
+        enrollButton.layer.cornerRadius = 2
+        enrollButton.setTitleColor(.white, for: .normal)
+        enrollButton.setTitle("ENROLL", for: .normal)
+        enrollButton.titleLabel?.font = .systemFont(ofSize: 12)
+        enrollButton.backgroundColor = .black
+        container.addSubview(enrollButton)
+        
+        removeButton.translatesAutoresizingMaskIntoConstraints = false
+        removeButton.layer.cornerRadius = 2
+        removeButton.setTitleColor(.courseGrabRuby, for: .normal)
+        removeButton.setTitle("REMOVE", for: .normal)
+        removeButton.titleLabel?.font = .systemFont(ofSize: 12)
+        removeButton.layer.borderColor = UIColor.courseGrabRuby.cgColor
+        removeButton.layer.borderWidth = 1
+        container.addSubview(removeButton)
+        
+        sectionLabel.translatesAutoresizingMaskIntoConstraints = false
+        sectionLabel.font = .boldSystemFont(ofSize: 14)
+        sectionLabel.textColor = .courseGrabGray
+        container.addSubview(sectionLabel)
+        
+        statusBadge.translatesAutoresizingMaskIntoConstraints = false
+        statusBadge.contentMode = .scaleAspectFit
+        container.addSubview(statusBadge)
+        
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.font = .systemFont(ofSize: 16)
+        titleLabel.numberOfLines = 0
+        container.addSubview(titleLabel)
+        
+        // setup constraints
+
+        container.snp.makeConstraints { make in
+            make.topMargin.equalTo(contentView).offset(6)
+            make.bottomMargin.equalTo(contentView).inset(6)
+            make.leadingMargin.equalTo(20)
+            make.trailingMargin.equalTo(contentView).inset(20)
+        }
+        
+        courseLabel.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).offset(12)
+            make.trailing.equalTo(container).inset(16)
+        }
+        
+        enrollButton.snp.makeConstraints { make in
+            make.top.equalTo(sectionLabel.snp.bottom).offset(12)
+            make.leading.equalTo(container.snp.centerX).offset(9)
+            make.height.equalTo(24)
+            make.trailing.equalTo(container).inset(16)
+            make.bottom.equalTo(container).inset(16)
```

Comment:
you can use `equalToSuperview()` instead of specifying `container` here.
you can also do `make.trailing.bottom.equalToSuperview().inset(16)`

Reply:
bump on this ^. since `enrollButton`'s superview is `container`, you should be able to do `make.trailing.bottom.equalToSuperview().inset(16)`

Reply:
What does "bump on this mean?" I'm pretty sure you're commenting on outdated code.

Reply:
yeah woops. for reference when i say "bump on this" i just mean to "look at this again"